Reference/Cache/index.md

```diff
@@ -29,13 +30,13 @@ The interface consists of the following methods. _Some of these methods may not
 * `void Refresh(Guid Id);` - this would invalidate or refresh a single cache for an object with the provided GUID id.
 * `void Remove(int Id);` - this would invalidate a single cache for an object with the provided INT id. In many cases Remove and Refresh perform the same operation but in some cases `Refresh` doesn't just remove/invalidate a cache entry, it might update it. `Remove` is specifically used to remove/invalidate a cache entry.
 
-There are 2 other base types of `ICacheRefresher` which are: 
+There are 2 other base types of `ICacheRefresher` which are:
 
-* `ICacheRefresher<T>` - this inherits from `ICacheRefresher` but provides strongly typed methods for cache invalidation too. This is useful when executing the method to invoke the cache refresher when you have the instance of the object already since this can avoid some overhead of re-looking the object back up.
-   * `void Refresh(T instance);` - this would invalidate/refresh a single cache for the specified object.
-   * `void Remove(T instance);` - this would invalidate a single cache for the specified object.
-* `IJsonCacheRefresher` - this inherits from `ICacheRefresher` but provides more flexibility if you need to invalidate cache based on more complex scenarios. 
-   * `void Refresh(string jsonPayload)` - Invalidates/refreshes any cache based on the information provided in the JSON. The JSON value is any value that is used when executing the method to invoke the cache refresher.
+* [`ICacheRefresher<T>`](cache-refresher-t.md) - this inherits from `ICacheRefresher` but provides strongly typed methods for cache invalidation too. This is useful when executing the method to invoke the cache refresher when you have the instance of the object already since this can avoid some overhead of re-looking the object back up.
```

Comment:
invoke the cache refresher, when you have the instance of the object already since this avoids the overhead of retrieving the object again.

Reply:
It's shannons explanation.  I think yours is better.  :-D